### PR TITLE
Feature migrate to capacitor 3

### PR DESCRIPTION
**Pull request by sultanmyrza**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201016280880500/1201944422184155) by [Unito](https://www.unito.io)
┆Created By: Tammy Yang

### CODE REVIEW

**Review by ruizehung — COMMENTED** [review]
Overall, very nice work @sultanmyrza ! The changes seem reasonable to me.

- I think we should use `TranslocoService` and add Mandarin translation for go-pro feature related text display in the app. e.g. The text in `go-pro-media-list-on-camera.component.html`, `go-pro.page.html` should have their correspoding Mandarin translation. I can help you with this. Let me know : )
- For the merge conflicts, I've addressed the profile page one in a comment below. The other 2 conflicts should be easy to resolve. I was able to locally merge this PR into the latest develop, build and run the app successfully.

File: src/app/features/settings/go-pro/go-pro-media-item-detail-on-camera/go-pro-media-item-detail-on-camera.component.ts [hunk 1]
```diff
@@ -46,42 +43,15 @@ export class GoProMediaItemDetailOnCameraComponent implements OnInit {
...
   async uploadToCapture() {
     const allowed = await this.allowUploadWithMobileInternet();
     if (allowed) {
       await this.startUploadToCapture();
 
-      if (
-        (await this.goProWiFiService.showTutorialForMobileDataOnlyApps()) ===
-        false
-      ) {
+      if (await this.goProWiFiService.showTutorialForMobileDataOnlyApps()) {
```
[the remark this answers is not in the document]
Comment (sultanmyrza) [reply src/app/features/settings/go-pro/go-pro-media-item-detail-on-camera/go-pro-media-item-detail-on-camera.component.ts:54]: @ruizehung Yes it's changin logic. I found a bug while testing so I changed logic to fix the bug. 🤗

File: package.json [hunk 2]
```diff
@@ -28,12 +28,25 @@
...
-    "@capacitor-community/bluetooth-le": "github:numbersprotocol/bluetooth-le#fix-complilation-error-on-ios-15",
-    "@capacitor-community/http": "^0.3.2",
-    "@capacitor-community/wifi": "git+https://github.com/digaus/community-capacitor-wifi.git",
-    "@capacitor/android": "https://gitpkg.now.sh/numbersprotocol/capacitor/android?release-2.4.7-range-request-fix",
-    "@capacitor/core": "~2.4.7",
-    "@capacitor/ios": "~2.4.7",
+    "@capacitor-community/bluetooth-le": "^1.7.0",
+    "@capacitor-community/http": "^1.4.1",
+    "@capacitor-community/wifi": "github:sultanmyrza/community-capacitor-wifi#capacitor3",
+    "@capacitor/android": "https://gitpkg.now.sh/sultanmyrza/capacitor/android?release-3.4.1-range-request-fix",
```
[the remark this answers is not in the document]
Comment (sultanmyrza) [reply package.json]: @ruizehung I moved dependencies to `numbersprotocol` 🤗 here is the commit [install capcitor wifi, android plugins from numbersprotocol repository](https://github.com/numbersprotocol/capture-lite/pull/1358/commits/29713f05aad0458f5c58c8a8eac8699e17bcada1)

File: src/app/features/profile/profile.page.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { Component } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
 import { MatSnackBar } from '@angular/material/snack-bar';
 import { ActivatedRoute, Router } from '@angular/router';
-import { Plugins } from '@capacitor/core';
+import { Clipboard } from '@capacitor/clipboard';
```
[the remark this answers is not in the document]
Comment (sultanmyrza) [reply src/app/features/profile/profile.page.ts]: @ruizehung Hope this commits resolves the issue. 🤗

[fix(profile.page.ts): remove unused imports](https://github.com/numbersprotocol/capture-lite/pull/1358/commits/6b60d54980500b93dbadd5cf71a61b35a389d723)

[fix(wallets.page.ts): import Browser, Clipboard from @capacitor package](https://github.com/numbersprotocol/capture-lite/pull/1358/commits/c78368d5bcf512317bf8f1b6c9a12759f8437fe5)

[the remark this answers is not in the document]
Comment (sultanmyrza) [reply src/app/features/profile/profile.page.ts]: @ruizehung also add Chinese translation https://github.com/numbersprotocol/capture-lite/pull/1358/commits/47efb195896dd12ed58283b9f380dcbfcac28872

**Review by ruizehung — COMMENTED** [review]
LGTM. @shc261392 do you have any comments or suggestions?

**Review by shc261392 — APPROVED** [review]

**Review by shc261392 — APPROVED** [review]

**Pull request merged**